KEXPPower/Public API/KEXPPower.swift

```diff
@@ -8,98 +8,68 @@
 
 import Foundation
 
-/// Available Stream Entry
-public typealias AvailableStream = (streamName: String, streamURL: URL)
-
-/// Archive Stream Entry
-public typealias ArchiveStream = (archiveBitRate: ArchiveBitRate, streamURL: URL)
-
-/// Available archive bit rates
-public enum ArchiveBitRate: String {
-    case thirtyTwo = "32"
-    case sixtyFour = "64"
-    case oneTwentyEight = "128"
-}
-
 /// Class used to configure KEXP networking
 public class KEXPPower {
+    /// Available archive bit rates
+    public enum ArchiveBitRate: Int {
```

Comment:
If we aren't fetching archive streams by bitrate, I think this should be named differently (also applies to the variable below)

Reply:
Renamed to KEXPPower.StreamingBitRate